builds/e2e/templates/connectivity-deploy.yaml

```diff
@@ -159,3 +160,20 @@ steps:
       E2E_parentHostname: ${{ parameters['connectivity.parentHostname'] }}
       E2E_parentEdgeDevice: ${{ parameters['connectivity.parentEdgeDevice'] }} 
       E2E_hostname: $(Hostname)
+
+  - pwsh: |
```

Comment:
A high level comment here about how the logs are generated and then renamed might improve readability.

Something that captures that the logs are generated in the test step above without the frequency suffix, then here we both copy it to a new place and change the name. 

Two questions I have about this are:
1. Can we name it correctly when the support bundle is generated (i.e. handle the rename in the trc bash script? We might need to pass in the network controller description to the test script for this to work.
2. Why do we need to copy it to a new place? Why can't PublishBuildArtifacts just take it from the original place it is first generated?

Reply:
I have a few followup comments
1. Passing the Network Controller Description has an issue because the script is shared with longhaul (Which doesnt have the network controller description defined). With the current way, a pipeline has freedom to choose what the artifact name needs to be.
2. I agree, I will update the step to copy directly from the initial location

Reply:
The function we are adding here is only being called for connectivity tests. Also, the script takes many parameters already that are either connectivity or longhaul specific. For example:
https://github.com/Azure/iotedge/blob/master/scripts/linux/trcE2ETest.sh#L244

It seems possible to add a parameter to the script for the network controller description then we can name it what we want. IME this indirection through yaml is the sort of thing that eventually leaks into other places and makes things harder to run locally. Not a big case here, but I'd prefer naming it correctly up-front.